.changeset/late-pens-arrive.md

```diff
@@ -0,0 +1,7 @@
+---
+'@pantheon-systems/nextjs-kit': minor
+'@pantheon-systems/next-wordpress-starter': minor
+'@pantheon-systems/next-drupal-starter': patch
```

Comment:
Did `next-drupal-starter` get pagination here or should this be a separate changeset?